modeshape-jcr/src/test/resources/config/cluster/repo-config-clustered-persistent-1.json

```diff
@@ -21,6 +21,9 @@
             "minimumBinarySizeInBytes": 4096
         }
     },
+    "clustering" : {
+        //use defaults
```

Comment:
I forget ... do we allow comments in JSON files? (Presumably, since this file must be used somewhere.)